types/__tests__/type-tests.ts

```diff
@@ -249,11 +249,11 @@ export async function testWithin() {
   container.queryAllByLabelText('Some label')
 
   container.getByText('Click me')
-  container.getByText<HTMLButtonElement>('Click me')
-  container.getAllByText<HTMLButtonElement>('Click me')
+  container.getByText('Click me')
+  container.getAllByText('Click me')
 
   await container.findByRole('button', {name: /click me/i})
-  container.getByRole<HTMLButtonElement>('button', {name: /click me/i})
+  container.getByRole('button', {name: /click me/i})
```

Comment:
Why did these change?

Reply:
That's somehow done by the precommit hook locally. I'll either try fix it locally or prep a commit at the end that reverts this change and skip the precommit there and create a task to track it.  

- [ ] Undo this changes